babel-watch.js

```diff
@@ -21,13 +22,13 @@ function collect(val, memo) {
   return memo;
 }
 
-program.option('-d, --debug [port]', 'Set debugger port')
```

Comment:
capital D is already used by `--disable-autowatch`, and node support `--debug` natively.
We can still use `babel-watch --debug`, and the option `--debug` will be passed to node.js.